apps/demos/Demos/HtmlEditor/MarkdownSupport/React/App.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useCallback, useState } from 'react';
+import HtmlEditor, { Toolbar, Item } from 'devextreme-react/html-editor';
+import { markup } from './data.ts';
+
+const headerValues = [false, 1, 2, 3, 4, 5];
+const headerOptions = {
+  inputAttr: {
+    'aria-label': 'Header',
+  },
+};
```

Comment:
[minor, no need to change]
I think it's better to place it in `data.ts` together with a markup